docs/source/routing.rst

```diff
@@ -595,3 +598,42 @@ internal or ``HTTP 302`` redirects.
 
   The :func:`redirect` utility, along with several other useful
   helpers, are documented in :ref:`pecan_core`.
+
+
+Determining the url for controller
```

Comment:
Use the same caps in the rest of the headlines - this looks weird with the other headlines, e.g.,

Determining the URL for Controller